taqueria-plugin-contract-types/src/generator/typescript-output.ts

```diff
@@ -223,6 +232,9 @@ ${tabs(indent)}}])`;
 		if (t.kind === `unknown`) {
 			return `unknown`;
 		}
+		if (t.kind === 'lambda') {
+			throw new GenerateApiError(`Don't know how to generate default value for lambda`, { t });
```

Comment:
Hmm, this warrants a new helper in the tas utils, so this should be something like this:

```
return `tas.intruction('')`
```

Also, this needs to be added to the tas code to convert a string to a MichelsonInstruction